README.md

```diff
@@ -29,7 +29,7 @@ For the many different ways to install Elixir,
 [see our installation instructions on the website](https://elixir-lang.org/install.html).
 To compile from source, you can follow the steps below.
 
-First, [install Erlang](https://elixir-lang.org/install.html#installing-erlang). Then clone this repository to your machine, compile and test it:
+First, [install Erlang](https://elixir-lang.org/install.html#installing-erlang). Then, clone this repository to your machine, compile and test it:
```

Comment:
I am not sure this is the correct form. Can we please revert the "Then," ones or find a reference that we can follow? :) Thank you.

Reply:
It is not that simple. Then can have various meanings.
In this particular case it a "sequencer" or an "adverb of sequence" (e.g. first, then, next, finaly) and it is used to describe an order of events. They have start the sentence, and they are followed by a comma.
Same with the  previous sentence that starts with "First, ".

In the conditional sentences when they are preceded by "If something. Then ...." I don't use comma.

Reply:
So I would rather prefer to replace the "then" for something less ambiguous, like "Next". Thoughts?

Reply:
"After that, " suits well in this particular case. "Next, " would fit if there is another event after it. But in other cases I am not sure they can replaced.
In any case, "First, ... . Then, ..." is not IMO.

Reply:
After that, next, finally, are all good to me.